src/main/java/com/moodybluez/enterprise/service/EntryServiceStub.java

```diff
@@ -16,8 +16,7 @@
 
     @Override
     public boolean saveEntry(Entry entry) throws Exception{
-        boolean result = entryDAO.saveEntry(entry);
-        return result;
+        return entryDAO.saveEntry(entry);
```

Comment:
This seemed to just be a small case of redundancy in the code. I believe this reduces some technical debt, and may make things look a little cleaner overall.  Here is a Stack Overflow article I read in regards to this: [SOURCE](https://stackoverflow.com/questions/42773027/local-variable-is-redundant-jetbrains-ides-e-g-intellij-idea)